.gitignore

```diff
@@ -12,3 +12,7 @@ node_modules/
 htmlcov/
 
 *.vscode/
+
+media/
+
+migrations/
```

Comment:
It'd give me peace of mind to make these a bit more specific

such as 
`dispatch/apps/**/migrations/`
and
`dispatch/tests/media`